gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/ResourceManager.java

```diff
@@ -263,4 +291,72 @@ public static ProjectListOption fields(ProjectField... fields) {
    * @throws ResourceManagerException upon failure
    */
   void undelete(String projectId);
+
+  /**
+   * Returns the IAM access control policy for the specified project. Returns null if the resource
+   * does not exist or if you do not have adequate permission to view the project.
+   *
+   * @see <a
+   * href="https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects/getIamPolicy">
+   * Resource Manager getIamPolicy</a>
+   * @throws ResourceManagerException upon failure
+   */
+  Policy getPolicy(String projectId);
+
+  /**
+   * Sets the IAM access control policy for the specified project. Replaces any existing policy. The
+   * following constraints apply:
+   * <ul>
+   * <li>Projects currently support only <I>user:{emailid}</I> and <I>serviceAccount:{emailid}</I>
```

Comment:
Will replace the capital "I"s with lowercase ones.  For the second point, I do wonder if it makes the javadoc (very slightly) more messy for people ready the source itself.  When I look up javadoc online, I usually don't see people adding the `</li>`, e.g. [in Guava](https://github.com/google/guava/search?utf8=%E2%9C%93&q=%3Cli%3E)